AABB_tree/doc/AABB_tree/Concepts/AABBTraits.h

```diff
@@ -145,21 +138,21 @@ typedef unspecified_type Compare_distance;
 
 /*!
 A functor object to compute closest point from the query on a primitive. Provides the operator:
-`Point_3 operator()(const Query& query, const Primitive& primitive, const Point_3 & closest);` which returns the closest point to `query`, among `closest` and all points of the primitive.
+`Point operator()(const Query& query, const Primitive& primitive, const Point & closest);` which returns the closest point to `query`, among `closest` and all points of the primitive.
```

Comment:
@sloriot this is already wording from 3D and we do not really understand what is meant here,

Reply:
closest is the current closest point to the query point. "all points of the primitive..." simply means the distance between `query` and the primitive

Reply:
Sorry, but even with the explanation it's not clear to me and I don't know how I would implement this.

We can put it aside because it's not really what the PR is about...

Reply:
"which returns the closest point to `query` on the primitive." Does it sound better? "Among" does not sound like the right choice of word to me.